examples/ui/grid.rs

```diff
@@ -0,0 +1,159 @@
+//! Demonstrates how the `AlignItems` and `JustifyContent` properties can be composed to layout text.
```

Comment:
This text seems to refer to a different example

Reply:
Fixed